test/test_base_metric.py

```diff
@@ -135,12 +135,12 @@ def test_get_metric_is_independent_from_metric_learner(estimator,
 
   # we fit the metric learner on it and then we compute the metric on some
   # points
-  model.fit(input_data, labels)
+  model.fit(*remove_y(model, input_data, labels))
```

Comment:
I don't know if it was a looked over or intentional, but `LSML` worked because it allows a second `weights` input. But in other tests, a `remove_y_quadruplet` was used.

Reply:
Nice catch - it was probably not intentional but it is good to fix it!

Reply:
I confirm, it was not intentional :p Good catch !